src/server/dbOps.js

```diff
@@ -185,6 +244,65 @@ class DataBaseRequest extends postgresOps.postgresOps {
 
         return accessTokenRecord.rows[0];
     };
+
+    /**
+     * Get a login_token record of a SSO user account for a referer origin.
+     *
+     * @param {Number} ssoUserAccountId - The sso user account id that the login token record is created for.
+     * @param {Object} refererOrigin - The referer origin where the SSO user sends the request from.
+     * @return {Object} An object consisting of the login token record.
+     */
+    async getLoginToken(ssoUserAccountId, refererOrigin) {
+        const loginTokenRecord = await this.runSql(`
+            SELECT * FROM login_token
+            WHERE sso_user_account_id = ${ssoUserAccountId}
+            AND referer_origin = '${refererOrigin}'
+        `);
+
+        return loginTokenRecord.rows[0];
+    };
+
+    /**
+     * Create a login_token record. Return an object containing the saved record.
+     *
+     * @param {Number} ssoUserAccountId - The sso user account id that the login token record is created for.
```

Comment:
These 4 are probably all {String}

Reply:
`ssoUserAccountId` is a number. It's a serial number auto generated by postgresDB as a primary key for the sso_user_account table.